templates/registration/login.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+  
+  {% if form.errors %}
+  <p>Your username and password didn't match. Please try again.</p>
+  {% endif %}
+
+  <h1>Login</h1>
+
+  {% if next %}
+    <p>Please login to see this page.</p>
+  {% endif %}
+
+  <form method="post" action="{% url 'login' %}">
+  {% csrf_token %}
+  {% load crispy_forms_tags %}
```

Comment:
The `{% load crispy_forms_tags %}` tag should be placed at the top of the template, before any other template tags, to ensure that the crispy forms tags are available when rendering the form.